11ty/definitions/optical-allyship.md

```diff
@@ -0,0 +1,28 @@
+---
+title: optical allyship
+slug: optical-allyship
+defined: true
+speech: noun
+flag:
+  level: tool
+  text: White Supremacy Tool
+reading:
+  - text: 'We are not interested in Optical Allyship (origin)'
+    href: https://www.instagram.com/p/BiPDZkbFJFY/?hl=en
+  - text: 'If You Want To Be Anti-Racist, This Non-Optical Allyship Guide Is Required Reading'
+    href: https://www.vogue.co.uk/arts-and-lifestyle/article/non-optical-ally-guide
+  - text: 'What Is Optical Allyship? 3 Ways To Be Actively Anti-Racist'
+    href: https://www.bustle.com/p/what-is-optical-allyship-3-ways-to-be-actively-anti-racist-22956518
+  - text: 'What is optical allyship? Ways you can support the BLM movement apart from spreading hashtags'
+    href: https://www.msn.com/en-ca/news/politics/what-is-optical-allyship-ways-you-can-support-the-blm-movement-apart-from-spreading-hashtags/ar-BB15cJgz
+---
+
+> allyship that only serves at the surface level to platform the 'ally,' it makes a statement but doesn’t go beneath the surface and is not aimed at breaking away from the systems of power that oppress.
```

Comment:
Can we rewrite this lead-in so it avoids a circular definition? Perhaps something like "when X happens,"

Reply:
I quoted the definition from the source/coiner. I don't feel comfortable redefining it.

Reply:
@autosponge I don't think we're seeking to define it, but instead adjust the introduction so that it fits the format of a dictionary entry. If you don't feel like you're able to do that, I'm happy to do it.

Reply:
an alternate approach is to leave the definition (perhaps prepending with "a form of allyship" and then to define allyship as a separate term. My main piece of feedback here is that it needs to fit into the dictionary definition framework. I hear your points about wanting to ensure the dictionary definition is accurate to the coiner and agree there.

Reply:
I added 'a form of allyship.' as the first line.

Reply:
```suggestion
> a form of allyship that only serves at the surface level to platform the 'ally,' it makes a statement but doesn’t go beneath the surface and is not aimed at breaking away from the systems of power that oppress.
```